tests/conftest.py

```diff
@@ -1,7 +1,14 @@
+import os
 import pytest
 from src.pycatcher import create_app
 from src.pycatcher.webapp import register_routes
 
+
+def pytest_configure(config):
```

Comment:
Setting the logger level to CRITICAL to suppress the INFO messages while running test cases.